demos/react/1.1.1/src/config.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * File: config.ts
+ * Project: React Example
+ * Created: Thursday, 1st September 2022
+ * Author: Mehdi Rashadatjou
+ * -----
+ * Copyright 2021 - 2022, ©Flowplayer
+ * -----
+ */
+
+export const DEMO_TOKEN =
+  "eyJraWQiOiJiRmFRNEdUam9lNVEiLCJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9.eyJjIjoie1wiYWNsXCI6NixcImlkXCI6XCJiRmFRNEdUam9lNVFcIixcImRvbWFpblwiOltcImJ1aWxkcy5mbG93cGxheWVyLmNvbVwiXX0iLCJpc3MiOiJGbG93cGxheWVyIn0.upfvSSPnB-v2ADHfbWG8ye9jDQhgwnMhZWQUqDS2DOLQbldCt9N8Atbq-gRm4GbqRRS7zoBFvvf6CgYWaV93nw";
```

Comment:
Is this token domain locked to some domain?

Reply:
The token is the same one as here https://github.com/flowplayer/react-flowplayer/blob/9ffbe86831ce3487f53e336424017b22b9cde504/demo/demo.tsx#L8